error: Could not connect to the database. connect ECONNREFUSED ::1:27017 {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"Single"},"stack":"MongooseServerSelectionError: connect ECONNREFUSED ::1:27017\n    at NativeConnection.Connection.openUri (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\Node_WorkSpace\\fundooNotes\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:17)\n    at _next (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:17:9)\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:7\n    at new Promise (<anonymous>)\n    at apply (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:14:12)\n    at database (D:\\Node_WorkSpace\\fundooNotes\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)\n    at Module._compile (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)\n    at Object.newLoader [as .js] (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1203:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1019:12)\n    at Function.runMain (node:internal/modules/run_main:128:12)\n    at Object.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\node\\src\\_babel-node.ts:228:12)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1422:10)\n    at Module.load (node:internal/modules/cjs/loader:1203:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1019:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-07-13T04:44:19.751Z"}
error: 
      status - 400
      message - Authorization token is required 
      url - /api/v1/users/getdata 
      method - GET 
      IP - ::1
     {"timestamp":"2024-07-13T04:44:28.233Z"}
error: Could not connect to the database. connect ECONNREFUSED ::1:27017 {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"Single"},"stack":"MongooseServerSelectionError: connect ECONNREFUSED ::1:27017\n    at NativeConnection.Connection.openUri (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\Node_WorkSpace\\fundooNotes\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:17)\n    at _next (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:17:9)\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:7\n    at new Promise (<anonymous>)\n    at apply (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:14:12)\n    at database (D:\\Node_WorkSpace\\fundooNotes\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)\n    at Module._compile (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)\n    at Object.newLoader [as .js] (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1203:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1019:12)\n    at Function.runMain (node:internal/modules/run_main:128:12)\n    at Object.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\node\\src\\_babel-node.ts:228:12)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1422:10)\n    at Module.load (node:internal/modules/cjs/loader:1203:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1019:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-07-13T04:50:26.580Z"}
error: 
    status - 500 
    message - CastError: Cast to ObjectId failed for value "getdata" (type string) at path "_id" for model "User"
    at model.Query.exec (D:\Node_WorkSpace\fundooNotes\node_modules\mongoose\lib\query.js:4498:21)
    at model.Query.Query.then (D:\Node_WorkSpace\fundooNotes\node_modules\mongoose\lib\query.js:4592:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/getdata 
    method - GET 
    IP - ::1
   {"timestamp":"2024-07-13T04:50:27.447Z"}
error: 
    status - 500 
    message - MongooseError: Operation `users.find()` buffering timed out after 10000ms
    at Timeout.<anonymous> (D:\Node_WorkSpace\fundooNotes\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:198:23)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7) 
    url - /api/v1/users/getdata 
    method - GET 
    IP - ::1
   {"timestamp":"2024-07-13T04:52:22.635Z"}
error: Could not connect to the database. connect ECONNREFUSED ::1:27017 {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"Single"},"stack":"MongooseServerSelectionError: connect ECONNREFUSED ::1:27017\n    at NativeConnection.Connection.openUri (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\Node_WorkSpace\\fundooNotes\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:17)\n    at _next (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:17:9)\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:7\n    at new Promise (<anonymous>)\n    at apply (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:14:12)\n    at database (D:\\Node_WorkSpace\\fundooNotes\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)\n    at Module._compile (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)\n    at Object.newLoader [as .js] (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1203:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1019:12)\n    at Function.runMain (node:internal/modules/run_main:128:12)\n    at Object.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\node\\src\\_babel-node.ts:228:12)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1422:10)\n    at Module.load (node:internal/modules/cjs/loader:1203:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1019:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-07-13T04:52:24.644Z"}
error: Could not connect to the database. connect ECONNREFUSED ::1:27017 {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"Single"},"stack":"MongooseServerSelectionError: connect ECONNREFUSED ::1:27017\n    at NativeConnection.Connection.openUri (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\Node_WorkSpace\\fundooNotes\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:17)\n    at _next (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:17:9)\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:7\n    at new Promise (<anonymous>)\n    at apply (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:14:12)\n    at database (D:\\Node_WorkSpace\\fundooNotes\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)\n    at Module._compile (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)\n    at Object.newLoader [as .js] (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1203:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1019:12)\n    at Function.runMain (node:internal/modules/run_main:128:12)\n    at Object.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\node\\src\\_babel-node.ts:228:12)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1422:10)\n    at Module.load (node:internal/modules/cjs/loader:1203:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1019:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-07-13T04:53:42.844Z"}
error: Could not connect to the database. connect ECONNREFUSED ::1:27017 {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"Single"},"stack":"MongooseServerSelectionError: connect ECONNREFUSED ::1:27017\n    at NativeConnection.Connection.openUri (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\Node_WorkSpace\\fundooNotes\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:17)\n    at _next (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:17:9)\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:7\n    at new Promise (<anonymous>)\n    at apply (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:14:12)\n    at database (D:\\Node_WorkSpace\\fundooNotes\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)\n    at Module._compile (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)\n    at Object.newLoader [as .js] (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1203:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1019:12)\n    at Function.runMain (node:internal/modules/run_main:128:12)\n    at Object.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\node\\src\\_babel-node.ts:228:12)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1422:10)\n    at Module.load (node:internal/modules/cjs/loader:1203:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1019:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-07-13T04:55:13.083Z"}
error: Could not connect to the database. connect ECONNREFUSED ::1:27017 {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"Single"},"stack":"MongooseServerSelectionError: connect ECONNREFUSED ::1:27017\n    at NativeConnection.Connection.openUri (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\Node_WorkSpace\\fundooNotes\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:17)\n    at _next (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:17:9)\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:7\n    at new Promise (<anonymous>)\n    at apply (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:14:12)\n    at database (D:\\Node_WorkSpace\\fundooNotes\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)\n    at Module._compile (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)\n    at Object.newLoader [as .js] (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1203:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1019:12)\n    at Function.runMain (node:internal/modules/run_main:128:12)\n    at Object.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\node\\src\\_babel-node.ts:228:12)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1422:10)\n    at Module.load (node:internal/modules/cjs/loader:1203:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1019:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-07-13T05:03:19.469Z"}
error: Could not connect to the database. connect ECONNREFUSED ::1:27017 {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"Single"},"stack":"MongooseServerSelectionError: connect ECONNREFUSED ::1:27017\n    at NativeConnection.Connection.openUri (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\Node_WorkSpace\\fundooNotes\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:17)\n    at _next (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:17:9)\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:7\n    at new Promise (<anonymous>)\n    at apply (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:14:12)\n    at database (D:\\Node_WorkSpace\\fundooNotes\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)\n    at Module._compile (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)\n    at Object.newLoader [as .js] (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1203:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1019:12)\n    at Function.runMain (node:internal/modules/run_main:128:12)\n    at Object.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\node\\src\\_babel-node.ts:228:12)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1422:10)\n    at Module.load (node:internal/modules/cjs/loader:1203:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1019:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-07-13T05:04:46.755Z"}
error: 
    status - 500 
    message - MongooseError: Operation `users.find()` buffering timed out after 10000ms
    at Timeout.<anonymous> (D:\Node_WorkSpace\fundooNotes\node_modules\mongoose\lib\drivers\node-mongodb-native\collection.js:198:23)
    at listOnTimeout (node:internal/timers:569:17)
    at processTimers (node:internal/timers:512:7) 
    url - /api/v1/users/getdata 
    method - GET 
    IP - ::1
   {"timestamp":"2024-07-13T05:09:21.073Z"}
error: Could not connect to the database. connect ECONNREFUSED ::1:27017 {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"Single"},"stack":"MongooseServerSelectionError: connect ECONNREFUSED ::1:27017\n    at NativeConnection.Connection.openUri (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\Node_WorkSpace\\fundooNotes\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:17)\n    at _next (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:17:9)\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:7\n    at new Promise (<anonymous>)\n    at apply (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:14:12)\n    at database (D:\\Node_WorkSpace\\fundooNotes\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)\n    at Module._compile (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)\n    at Object.newLoader [as .js] (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1203:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1019:12)\n    at Function.runMain (node:internal/modules/run_main:128:12)\n    at Object.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\node\\src\\_babel-node.ts:228:12)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1422:10)\n    at Module.load (node:internal/modules/cjs/loader:1203:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1019:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-07-13T05:18:59.029Z"}
error: Could not connect to the database. connect ECONNREFUSED ::1:27017 {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"Single"},"stack":"MongooseServerSelectionError: connect ECONNREFUSED ::1:27017\n    at NativeConnection.Connection.openUri (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\Node_WorkSpace\\fundooNotes\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:17)\n    at _next (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:17:9)\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:7\n    at new Promise (<anonymous>)\n    at apply (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:14:12)\n    at database (D:\\Node_WorkSpace\\fundooNotes\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)\n    at Module._compile (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)\n    at Object.newLoader [as .js] (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1203:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1019:12)\n    at Function.runMain (node:internal/modules/run_main:128:12)\n    at Object.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\node\\src\\_babel-node.ts:228:12)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1422:10)\n    at Module.load (node:internal/modules/cjs/loader:1203:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1019:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-07-13T05:23:35.337Z"}
error: Could not connect to the database. connect ECONNREFUSED ::1:27017 {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"Single"},"stack":"MongooseServerSelectionError: connect ECONNREFUSED ::1:27017\n    at NativeConnection.Connection.openUri (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\Node_WorkSpace\\fundooNotes\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:17)\n    at _next (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:17:9)\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:7\n    at new Promise (<anonymous>)\n    at apply (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:14:12)\n    at database (D:\\Node_WorkSpace\\fundooNotes\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)\n    at Module._compile (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)\n    at Object.newLoader [as .js] (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1203:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1019:12)\n    at Function.runMain (node:internal/modules/run_main:128:12)\n    at Object.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\node\\src\\_babel-node.ts:228:12)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1422:10)\n    at Module.load (node:internal/modules/cjs/loader:1203:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1019:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-07-13T05:37:43.097Z"}
error: Could not connect to the database. connect ECONNREFUSED ::1:27017 {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"Single"},"stack":"MongooseServerSelectionError: connect ECONNREFUSED ::1:27017\n    at NativeConnection.Connection.openUri (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\Node_WorkSpace\\fundooNotes\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:17)\n    at _next (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:17:9)\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:7\n    at new Promise (<anonymous>)\n    at apply (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:14:12)\n    at database (D:\\Node_WorkSpace\\fundooNotes\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)\n    at Module._compile (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)\n    at Object.newLoader [as .js] (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1203:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1019:12)\n    at Function.runMain (node:internal/modules/run_main:128:12)\n    at Object.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\node\\src\\_babel-node.ts:228:12)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1422:10)\n    at Module.load (node:internal/modules/cjs/loader:1203:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1019:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-07-13T05:38:41.124Z"}
error: Could not connect to the database. connect ECONNREFUSED ::1:27017 {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"Single"},"stack":"MongooseServerSelectionError: connect ECONNREFUSED ::1:27017\n    at NativeConnection.Connection.openUri (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\Node_WorkSpace\\fundooNotes\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:17)\n    at _next (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:17:9)\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:7\n    at new Promise (<anonymous>)\n    at apply (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:14:12)\n    at database (D:\\Node_WorkSpace\\fundooNotes\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)\n    at Module._compile (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)\n    at Object.newLoader [as .js] (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1203:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1019:12)\n    at Function.runMain (node:internal/modules/run_main:128:12)\n    at Object.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\node\\src\\_babel-node.ts:228:12)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1422:10)\n    at Module.load (node:internal/modules/cjs/loader:1203:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1019:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-07-13T05:45:40.845Z"}
error: Could not connect to the database. connect ECONNREFUSED ::1:27017 {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"Single"},"stack":"MongooseServerSelectionError: connect ECONNREFUSED ::1:27017\n    at NativeConnection.Connection.openUri (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\Node_WorkSpace\\fundooNotes\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:17)\n    at _next (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:17:9)\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:7\n    at new Promise (<anonymous>)\n    at apply (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:14:12)\n    at database (D:\\Node_WorkSpace\\fundooNotes\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)\n    at Module._compile (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)\n    at Object.newLoader [as .js] (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1203:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1019:12)\n    at Function.runMain (node:internal/modules/run_main:128:12)\n    at Object.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\node\\src\\_babel-node.ts:228:12)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1422:10)\n    at Module.load (node:internal/modules/cjs/loader:1203:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1019:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-07-13T05:50:06.930Z"}
error: Could not connect to the database. connect ECONNREFUSED ::1:27017 {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"Single"},"stack":"MongooseServerSelectionError: connect ECONNREFUSED ::1:27017\n    at NativeConnection.Connection.openUri (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\Node_WorkSpace\\fundooNotes\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:17)\n    at _next (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:17:9)\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:7\n    at new Promise (<anonymous>)\n    at apply (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:14:12)\n    at database (D:\\Node_WorkSpace\\fundooNotes\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)\n    at Module._compile (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)\n    at Object.newLoader [as .js] (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1203:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1019:12)\n    at Function.runMain (node:internal/modules/run_main:128:12)\n    at Object.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\node\\src\\_babel-node.ts:228:12)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1422:10)\n    at Module.load (node:internal/modules/cjs/loader:1203:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1019:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-07-13T05:51:07.966Z"}
error: Could not connect to the database. connect ECONNREFUSED ::1:27017 {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"Single"},"stack":"MongooseServerSelectionError: connect ECONNREFUSED ::1:27017\n    at NativeConnection.Connection.openUri (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:351:10\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (D:\\Node_WorkSpace\\fundooNotes\\src\\config\\/database.js:12:20)\n    at tryCatch (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:17)\n    at _next (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:17:9)\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:22:7\n    at new Promise (<anonymous>)\n    at apply (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:14:12)\n    at database (D:\\Node_WorkSpace\\fundooNotes\\src\\config\\/database.js:4:15)\n    at Object.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\src\\/index.js:30:9)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)\n    at Module._compile (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\pirates\\lib\\index.js:117:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)\n    at Object.newLoader [as .js] (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\pirates\\lib\\index.js:121:7)\n    at Module.load (node:internal/modules/cjs/loader:1203:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1019:12)\n    at Function.runMain (node:internal/modules/run_main:128:12)\n    at Object.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\node\\src\\_babel-node.ts:228:12)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1422:10)\n    at Module.load (node:internal/modules/cjs/loader:1203:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:1019:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12)\n    at node:internal/main/run_main_module:28:49","timestamp":"2024-07-13T05:52:03.197Z"}
error: 
    status - 500 
    message - ReferenceError: users is not defined
    at getUsers (D:\Node_WorkSpace\fundooNotes\src\controllers\/user.controller.js:12:24)
    at Layer.handle [as handle_request] (D:\Node_WorkSpace\fundooNotes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Node_WorkSpace\fundooNotes\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\Node_WorkSpace\fundooNotes\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\Node_WorkSpace\fundooNotes\node_modules\express\lib\router\layer.js:95:5)
    at D:\Node_WorkSpace\fundooNotes\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\Node_WorkSpace\fundooNotes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Node_WorkSpace\fundooNotes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Node_WorkSpace\fundooNotes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Node_WorkSpace\fundooNotes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Node_WorkSpace\fundooNotes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Node_WorkSpace\fundooNotes\node_modules\express\lib\router\index.js:328:13)
    at D:\Node_WorkSpace\fundooNotes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Node_WorkSpace\fundooNotes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Node_WorkSpace\fundooNotes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Node_WorkSpace\fundooNotes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Node_WorkSpace\fundooNotes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Node_WorkSpace\fundooNotes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Node_WorkSpace\fundooNotes\node_modules\express\lib\router\index.js:328:13)
    at D:\Node_WorkSpace\fundooNotes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Node_WorkSpace\fundooNotes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Node_WorkSpace\fundooNotes\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\Node_WorkSpace\fundooNotes\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\Node_WorkSpace\fundooNotes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Node_WorkSpace\fundooNotes\node_modules\express\lib\router\index.js:328:13)
    at D:\Node_WorkSpace\fundooNotes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Node_WorkSpace\fundooNotes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Node_WorkSpace\fundooNotes\node_modules\express\lib\router\index.js:280:10)
    at D:\Node_WorkSpace\fundooNotes\node_modules\body-parser\lib\read.js:137:5
    at AsyncResource.runInAsyncScope (node:async_hooks:203:9)
    at invokeCallback (D:\Node_WorkSpace\fundooNotes\node_modules\raw-body\index.js:238:16)
    at done (D:\Node_WorkSpace\fundooNotes\node_modules\raw-body\index.js:227:7)
    at IncomingMessage.onEnd (D:\Node_WorkSpace\fundooNotes\node_modules\raw-body\index.js:287:7)
    at IncomingMessage.emit (node:events:517:28)
    at IncomingMessage.emit (node:domain:489:12)
    at endReadableNT (node:internal/streams/readable:1400:12)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) 
    url - /api/v1/users 
    method - GET 
    IP - ::1
   {"timestamp":"2024-07-13T06:44:10.675Z"}
error: uncaughtException: HttpStatus is not defined
ReferenceError: HttpStatus is not defined
    at _callee$ (D:\Node_WorkSpace\fundooNotes\src\controllers\/user.controller.js:5:16)
    at tryCatch (D:\Node_WorkSpace\fundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Node_WorkSpace\fundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Node_WorkSpace\fundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Node_WorkSpace\fundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Node_WorkSpace\fundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) {"date":"Sat Jul 13 2024 12:14:56 GMT+0530 (India Standard Time)","error":{},"exception":true,"os":{"loadavg":[0,0,0],"uptime":8002.203},"process":{"argv":["node","D:\\Node_WorkSpace\\fundooNotes\\src\\index.js"],"cwd":"D:\\Node_WorkSpace\\fundooNotes","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":18930004,"external":20860884,"heapTotal":87650304,"heapUsed":52926776,"rss":128262144},"pid":15160,"uid":null,"version":"v18.20.4"},"stack":"ReferenceError: HttpStatus is not defined\n    at _callee$ (D:\\Node_WorkSpace\\fundooNotes\\src\\controllers\\/user.controller.js:5:16)\n    at tryCatch (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.next (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:17)\n    at _next (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:17:9)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2024-07-13T06:44:56.312Z","trace":[{"column":16,"file":"D:\\Node_WorkSpace\\fundooNotes\\src\\controllers\\/user.controller.js","function":"_callee$","line":5,"method":null,"native":false},{"column":16,"file":"D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js","function":"tryCatch","line":45,"method":null,"native":false},{"column":17,"file":"D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js","function":null,"line":133,"method":null,"native":false},{"column":21,"file":"D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js","function":"Generator.next","line":74,"method":"next","native":false},{"column":17,"file":"D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js","function":"asyncGeneratorStep","line":3,"method":null,"native":false},{"column":9,"file":"D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js","function":"_next","line":17,"method":null,"native":false},{"column":5,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":95,"method":null,"native":false}]}
error: uncaughtException: E11000 duplicate key error collection: myDB.users index: email_1 dup key: { email: "ria@gmail.com" }
MongoError: E11000 duplicate key error collection: myDB.users index: email_1 dup key: { email: "ria@gmail.com" }
    at Function.create (D:\Node_WorkSpace\fundooNotes\node_modules\mongodb\lib\core\error.js:59:12)
    at toError (D:\Node_WorkSpace\fundooNotes\node_modules\mongodb\lib\utils.js:130:22)
    at D:\Node_WorkSpace\fundooNotes\node_modules\mongodb\lib\operations\common_functions.js:258:39
    at handler (D:\Node_WorkSpace\fundooNotes\node_modules\mongodb\lib\core\sdam\topology.js:961:24)
    at D:\Node_WorkSpace\fundooNotes\node_modules\mongodb\lib\cmap\connection_pool.js:352:13
    at handleOperationResult (D:\Node_WorkSpace\fundooNotes\node_modules\mongodb\lib\core\sdam\server.js:567:5)
    at MessageStream.messageHandler (D:\Node_WorkSpace\fundooNotes\node_modules\mongodb\lib\cmap\connection.js:308:5)
    at MessageStream.emit (node:events:517:28)
    at MessageStream.emit (node:domain:489:12)
    at processIncomingData (D:\Node_WorkSpace\fundooNotes\node_modules\mongodb\lib\cmap\message_stream.js:144:12)
    at MessageStream._write (D:\Node_WorkSpace\fundooNotes\node_modules\mongodb\lib\cmap\message_stream.js:42:5)
    at writeOrBuffer (node:internal/streams/writable:392:12)
    at _write (node:internal/streams/writable:333:10)
    at MessageStream.Writable.write (node:internal/streams/writable:337:10)
    at Socket.ondata (node:internal/streams/readable:809:22)
    at Socket.emit (node:events:517:28)
    at Socket.emit (node:domain:489:12)
    at addChunk (node:internal/streams/readable:368:12)
    at readableAddChunk (node:internal/streams/readable:341:9)
    at Socket.Readable.push (node:internal/streams/readable:278:10)
    at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {"date":"Sat Jul 13 2024 12:16:32 GMT+0530 (India Standard Time)","error":{"code":11000,"driver":true,"index":0,"keyPattern":{"email":1},"keyValue":{"email":"ria@gmail.com"},"name":"MongoError"},"exception":true,"os":{"loadavg":[0,0,0],"uptime":8098.046},"process":{"argv":["node","D:\\Node_WorkSpace\\fundooNotes\\src\\index.js"],"cwd":"D:\\Node_WorkSpace\\fundooNotes","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":18423040,"external":20351304,"heapTotal":49508352,"heapUsed":45803368,"rss":90148864},"pid":2936,"uid":null,"version":"v18.20.4"},"stack":"MongoError: E11000 duplicate key error collection: myDB.users index: email_1 dup key: { email: \"ria@gmail.com\" }\n    at Function.create (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongodb\\lib\\core\\error.js:59:12)\n    at toError (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongodb\\lib\\utils.js:130:22)\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongodb\\lib\\operations\\common_functions.js:258:39\n    at handler (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongodb\\lib\\core\\sdam\\topology.js:961:24)\n    at D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongodb\\lib\\cmap\\connection_pool.js:352:13\n    at handleOperationResult (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongodb\\lib\\core\\sdam\\server.js:567:5)\n    at MessageStream.messageHandler (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongodb\\lib\\cmap\\connection.js:308:5)\n    at MessageStream.emit (node:events:517:28)\n    at MessageStream.emit (node:domain:489:12)\n    at processIncomingData (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:144:12)\n    at MessageStream._write (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongodb\\lib\\cmap\\message_stream.js:42:5)\n    at writeOrBuffer (node:internal/streams/writable:392:12)\n    at _write (node:internal/streams/writable:333:10)\n    at MessageStream.Writable.write (node:internal/streams/writable:337:10)\n    at Socket.ondata (node:internal/streams/readable:809:22)\n    at Socket.emit (node:events:517:28)\n    at Socket.emit (node:domain:489:12)\n    at addChunk (node:internal/streams/readable:368:12)\n    at readableAddChunk (node:internal/streams/readable:341:9)\n    at Socket.Readable.push (node:internal/streams/readable:278:10)\n    at TCP.onStreamRead (node:internal/stream_base_commons:190:23)","timestamp":"2024-07-13T06:46:32.147Z","trace":[{"column":12,"file":"D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongodb\\lib\\core\\error.js","function":"Function.create","line":59,"method":"create","native":false},{"column":22,"file":"D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongodb\\lib\\utils.js","function":"toError","line":130,"method":null,"native":false},{"column":39,"file":"D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongodb\\lib\\operations\\common_functions.js","function":null,"line":258,"method":null,"native":false},{"column":24,"file":"D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongodb\\lib\\core\\sdam\\topology.js","function":"handler","line":961,"method":null,"native":false},{"column":13,"file":"D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongodb\\lib\\cmap\\connection_pool.js","function":null,"line":352,"method":null,"native":false},{"column":5,"file":"D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongodb\\lib\\core\\sdam\\server.js","function":"handleOperationResult","line":567,"method":null,"native":false},{"column":5,"file":"D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongodb\\lib\\cmap\\connection.js","function":"MessageStream.messageHandler","line":308,"method":"messageHandler","native":false},{"column":28,"file":"node:events","function":"MessageStream.emit","line":517,"method":"emit","native":false},{"column":12,"file":"node:domain","function":"MessageStream.emit","line":489,"method":"emit","native":false},{"column":12,"file":"D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongodb\\lib\\cmap\\message_stream.js","function":"processIncomingData","line":144,"method":null,"native":false},{"column":5,"file":"D:\\Node_WorkSpace\\fundooNotes\\node_modules\\mongodb\\lib\\cmap\\message_stream.js","function":"MessageStream._write","line":42,"method":"_write","native":false},{"column":12,"file":"node:internal/streams/writable","function":"writeOrBuffer","line":392,"method":null,"native":false},{"column":10,"file":"node:internal/streams/writable","function":"_write","line":333,"method":null,"native":false},{"column":10,"file":"node:internal/streams/writable","function":"MessageStream.Writable.write","line":337,"method":"write","native":false},{"column":22,"file":"node:internal/streams/readable","function":"Socket.ondata","line":809,"method":"ondata","native":false},{"column":28,"file":"node:events","function":"Socket.emit","line":517,"method":"emit","native":false},{"column":12,"file":"node:domain","function":"Socket.emit","line":489,"method":"emit","native":false},{"column":12,"file":"node:internal/streams/readable","function":"addChunk","line":368,"method":null,"native":false},{"column":9,"file":"node:internal/streams/readable","function":"readableAddChunk","line":341,"method":null,"native":false},{"column":10,"file":"node:internal/streams/readable","function":"Socket.Readable.push","line":278,"method":"push","native":false},{"column":23,"file":"node:internal/stream_base_commons","function":"TCP.onStreamRead","line":190,"method":"onStreamRead","native":false}]}
error: uncaughtException: HttpStatus is not defined
ReferenceError: HttpStatus is not defined
    at _callee2$ (D:\Node_WorkSpace\fundooNotes\src\controllers\/user.controller.js:29:16)
    at tryCatch (D:\Node_WorkSpace\fundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Node_WorkSpace\fundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.throw (D:\Node_WorkSpace\fundooNotes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Node_WorkSpace\fundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _throw (D:\Node_WorkSpace\fundooNotes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:20:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) {"date":"Mon Jul 15 2024 16:42:56 GMT+0530 (India Standard Time)","error":{},"exception":true,"os":{"loadavg":[0,0,0],"uptime":196881.968},"process":{"argv":["node","D:\\Node_WorkSpace\\fundooNotes\\src\\index.js"],"cwd":"D:\\Node_WorkSpace\\fundooNotes","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":18298002,"external":20226266,"heapTotal":50032640,"heapUsed":45149464,"rss":89174016},"pid":22488,"uid":null,"version":"v18.20.4"},"stack":"ReferenceError: HttpStatus is not defined\n    at _callee2$ (D:\\Node_WorkSpace\\fundooNotes\\src\\controllers\\/user.controller.js:29:16)\n    at tryCatch (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:45:16)\n    at Generator.<anonymous> (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:133:17)\n    at Generator.throw (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:74:21)\n    at asyncGeneratorStep (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:17)\n    at _throw (D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:20:9)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)","timestamp":"2024-07-15T11:12:56.075Z","trace":[{"column":16,"file":"D:\\Node_WorkSpace\\fundooNotes\\src\\controllers\\/user.controller.js","function":"_callee2$","line":29,"method":null,"native":false},{"column":16,"file":"D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js","function":"tryCatch","line":45,"method":null,"native":false},{"column":17,"file":"D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js","function":null,"line":133,"method":null,"native":false},{"column":21,"file":"D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js","function":"Generator.throw","line":74,"method":"throw","native":false},{"column":17,"file":"D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js","function":"asyncGeneratorStep","line":3,"method":null,"native":false},{"column":9,"file":"D:\\Node_WorkSpace\\fundooNotes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js","function":"_throw","line":20,"method":null,"native":false},{"column":5,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":95,"method":null,"native":false}]}
error: 
    status - 500 
    message - SyntaxError: Unexpected string in JSON at position 38
    at JSON.parse (<anonymous>)
    at parse (D:\Node_WorkSpace\fundooNotes\node_modules\body-parser\lib\types\json.js:92:19)
    at D:\Node_WorkSpace\fundooNotes\node_modules\body-parser\lib\read.js:128:18
    at AsyncResource.runInAsyncScope (node:async_hooks:203:9)
    at invokeCallback (D:\Node_WorkSpace\fundooNotes\node_modules\raw-body\index.js:238:16)
    at done (D:\Node_WorkSpace\fundooNotes\node_modules\raw-body\index.js:227:7)
    at IncomingMessage.onEnd (D:\Node_WorkSpace\fundooNotes\node_modules\raw-body\index.js:287:7)
    at IncomingMessage.emit (node:events:517:28)
    at IncomingMessage.emit (node:domain:489:12)
    at endReadableNT (node:internal/streams/readable:1400:12)
    at processTicksAndRejections (node:internal/process/task_queues:82:21) 
    url - /api/v1/users/get 
    method - GET 
    IP - ::1
   {"timestamp":"2024-07-15T11:24:32.020Z"}
